_episodes_rmd/13-dplyr.Rmd

```diff
@@ -181,6 +181,17 @@ even better.
 > >lifeExp_bycountry <- gapminder %>%
 > >    group_by(country) %>%
 > >    summarize(mean_lifeExp=mean(lifeExp))
+> >
+> >lifeExp_bycountry$country[which.max(lifeExp_bycountry$mean_lifeExp)]
+> >lifeExp_bycountry$country[which.min(lifeExp_bycountry$mean_lifeExp)]
```

Comment:
Might take these out since we're focusing on the dplyr approach in this episode.